torch/onnx/_internal/exporter/_fx_passes.py

```diff
@@ -15,13 +15,20 @@ def decompose_with_registry(
 
     This function is needed so it shows clearly on the profiler results.
     """
+    from torch._decomp import decomp_table_to_post_autograd_aten
```

Comment:
Is the delayed import required, or would it be possible to import the _decomp module on top and call _decomp.decomp_table_to_… below?